wandb/sdk/launch/runner/kubernetes_runner.py

```diff
@@ -99,6 +108,110 @@ def __init__(
         self._fail_count = 0
         self.pod_names = pod_names
         self.secret = secret
+        self._status_lock = Lock()
+        self._status = Status("running")
```

Comment:
doesnt feel right to start the status as `Running`

Reply:
We have a "starting" state, nothing specifically is done by it yet